app/scripts/modules/amazon/src/loadBalancer/configure/application/ALBListeners.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { $q } from 'ngimport';
 import { SortableContainer, SortableElement, SortableHandle, arrayMove, SortEnd } from 'react-sortable-hoc';
-import { difference, flatten, get, uniq } from 'lodash';
+import { difference, flatten, get, some, uniq } from 'lodash';
```

Comment:
This file is way too long.  We could probably extract some child components or something

Reply:
Thanks for volunteering!